test/unity/toolkits/object_detection/test_object_detector.cxx

```diff
@@ -719,6 +713,9 @@ BOOST_AUTO_TEST_CASE(test_object_detector_auto_split) {
   // used as a normalizer.
   gl_sframe data({{"ignored", gl_sarray::from_sequence(0, test_num_examples)}});
 
+  data_iterator::parameters iterator_params; 
+  iterator_params.data = std::move(data);  
```

Comment:
Don't use std::move here, since `data` is still used below.

Reply:
Wait, where is `iterator_params` used? Do we still need it?

Reply:
Good catch, we don't need this anymore